.github/workflows/durable.yml

```diff
@@ -0,0 +1,105 @@
+name: Run Durable tests
+
+on:
+  pull_request:
+    # TODO: Update after hermetic tests are merged
+    branches: [ "main", "feat/hermetic" ]
+  merge_group:
+    branches: [ "main" ]
+  schedule:
+    # Run every midnight UTC
+    - cron: "0 0 * * *"
+  workflow_dispatch: # manually triggered
+    inputs:
+      environment:
+        description: 'Environment name (one of: dev, qa, tnet, prod)'
+        required: true
+        default: 'qa'
+
+env:
+  AWS_REGION: ${{ secrets.AWS_REGION }}
+  AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+  AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+
+jobs:
+  publish-suite:
+    runs-on: ubuntu-latest
+    outputs:
+      build_tag: ${{ steps.generate-build-tag.outputs.build_tag }}
+    steps:
+    -
+      uses: actions/checkout@v3
+    -
+      name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v2
+    -
+      name: Login to Public ECR
+      uses: docker/login-action@v2
+      with:
+        registry: public.ecr.aws
+        username: ${{ env.AWS_ACCESS_KEY_ID }}
+        password: ${{ env.AWS_SECRET_ACCESS_KEY }}
+      env:
+        AWS_REGION: us-east-1
+    -
+      name: Generate build tag
+      id: generate-build-tag
+      run: |
+        # Choose unique name for this build
+        BUILD_TAG="$(echo ${{ github.sha }} | head -c 8)-${{ github.run_id }}"
+        echo "Build tag:"
+        echo ${BUILD_TAG}
+        echo "build_tag=${BUILD_TAG}" >> $GITHUB_OUTPUT
+        cat $GITHUB_OUTPUT
+    -
+      name: Build and Publish
+      run: make BUILD_TAG=${{ steps.generate-build-tag.outputs.build_tag }} publish-suite
+
+  run-tests:
+    name: Test
+    runs-on: ubuntu-latest
+    needs:
+      - publish-suite
+    strategy:
+      fail-fast: false
+      matrix:
+        network: [dev, qa, tnet, prod]
+        include:
+          - network: dev
+            composedb_urls: DEV_COMPOSEDB_URLS
```

Comment:
Where are the `DEV_COMPOSEDB_URLS` and related env vars defined?

Reply:
In the repo secrets.

Reply:
ok, do they need to be secret? I'd like the ability to run the durable tests from a local checkout of the repo. That way we can easily trigger them to run and debug any issues. Thoughts?

Reply:
Hmm, traditionally, we've kept them in secrets, but they're pretty much our worst-kept secret. These URLs were never fully secret to begin with (they're public, just not advertised) and the DNS names have always been exposed in the swarm [peerlist](https://github.com/ceramicnetwork/peerlist/) anyway.

I'd be fine putting them in the code.